CONTRIBUTION

```diff
@@ -0,0 +1,26 @@
+Making Contributions
+
+Cothority is an Open Source program with many contributors (listed in the AUTHORS
+files), so we actively seek contributions (help on the Cothority users list,
+documentation, source code, ideas, …).  All contributions help us make a
+better product.
+
+All contributions without an explicit copyright statement or a CLAI/CLAC (see below)
+are assumed to be covered under a AGPL 2-Clause license as described in the
+file LICENSE.
+
+Developers who have contributed significant changes to the Cothority code must sign
+a Contributor License Agreement (CLAI), eventually also a CLAC if they work for
```

Comment:
That's a good point. For the moment we only have contributions from other labs - will they accept to sign the CLA[IC]? 

Now that we have an actual text we can ask people what they think about. Please write down your concerns, so we can discuss them at the next meeting.